flexible_event_config.md

```diff
@@ -212,15 +217,46 @@ Triggers could be registered with the value field set, which are summed up and b
 { "event_trigger_data": [{"trigger_data": "0", "value": 4}] }
 ```
 
-The values are summed (to 8) and reported as bucket [5, 9]
+The values are summed (to 8) and reported in the following reports after 7 days + 1 hour:
```

Comment:
There are some ongoing work to set different delay for the reporting time. Do we need to make some change?

Reply:
The report time is determined by the `end_times` field here so it shouldn't be related to any changes in defaults, etc.